test/series.js

```diff
@@ -34,6 +34,42 @@ describe('series', function() {
     done();
   });
 
+  it('should throw on empty object on registered tasks array', function(done) {
+    function fail() {
+      taker.series(['test1', 'test2', 'test3', {}]);
```

Comment:
You can add another test for `taker.series('test1', 'test2', 'test3', {});` (that's without the array).